.azure-pipelines/common/test.yml

```diff
@@ -1,47 +1,48 @@
 steps:
-- script: |
```

Comment:
Can you revert the whitespace changes in this file?

Reply:
Ugh my stupid auto-formatter...